CITATION.cff

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+# This CITATION.cff file was initially generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: alchemlyb
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - email: dotsdl@gmail.com
+    given-names: David L.
+    family-names: Dotson
+    orcid: 'https://orcid.org/0000-0001-5879-2942'
+    affiliation: Datryllic LLC
```

Comment:
@dotsdl are you ok with this entry, in particular your email address here?

Reply:
Yes, this email works for this purpose! Thanks so much for leading this!

Reply:
Actually, could we swap this out for `david@datryllic.com` instead? Thank you!

Reply:
of course! — done with your suggestion